api.rb

```diff
@@ -0,0 +1,22 @@
+require 'sinatra'
+require 'jwt'
+require_relative 'lib/notify'
+
+class MyApi < Sinatra::Base
+  before do
+    halt 401 if request.env['HTTP_AUTHORIZATION'].nil?
+    acces_token = request.env['HTTP_AUTHORIZATION'].slice(7..-1)
+    JWT.decode acces_token, public_key, true, { algorithm: 'RS256' }
+  rescue StandardError
```

Comment:
Can we rescue more a specific error? What does `JWT.decode` raise if it's invalid?

Reply:
@tijmenb I had that, but it was going inside JWT.DecodeError, for SignatureExpired as well..
is it because of ordering? I want to anyway throw `403` for any exceptions while parsing the token.